### ISSUE TITLE
Allow for alternate av1 encoders?

### ISSUE THREAD

**Issue by andrew-kennedy** [HaveAGitGat/Tdarr_Plugins#541]
I'd like to use libaom-av1 instead of libsvt-av1, is there a fairly easy way to modify the setVideoCodec plugin to do this or perhaps an example of how to do this as a custom argument in the most flexible way to be stacked as part of a large ffmpeg command?

**Comment by HaveAGitGat** [comment]
Yeah that's a tricky one as currently not setup that way, will need to think about a solution for this. Quick solution would be to not use the `set video encoder` plugin and just use the custom arguments plugin as you mentioned. Then in it set:

`-c:v libaom-av1` 

Has that worked for you?
